packages/@expo/cli/src/lint/lintAsync.ts

```diff
@@ -0,0 +1,115 @@
+import JsonFile from '@expo/json-file';
+import * as PackageManager from '@expo/package-manager';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs/promises';
+import path from 'path';
+
+import { Log } from '../log';
+import { selectAsync } from '../utils/prompts';
+
+const WITH_PRETTIER = `module.exports = {
+  root: true,
+  extends: ["universe/native", "prettier"],
+  plugins: ["prettier", "expo"],
+  rules: {
+    "prettier/prettier": ["warn"],
+    "expo/no-env-var-destructuring": ["error"],
+    "expo/no-dynamic-env-var": ["error"],
+  },
+};
+
+`;
```

Comment:
I think setting up prettier would require more configuration. For example, you'll also need a prettier config file: https://prettier.io/docs/en/configuration.html

Maybe it's nicer to add prettier as "mutating the base eslint config", as in, a function that just appends things to `plugins` and generates the prettier config if not already configured?

Reply:
Also, might be good to keep prettier out of scope for now, and merge the initial version without.

Reply:
Last thing I can think of, make sure that every property within this eslint config is required. I usually struggle with adding or not adding `root` (no idea how this would work in users projects / monorepos etc).

Reply:
Good point about the `.prettierrc`, I forgot. The eslint config I'm using for testing already has it included so it was hard to verify. I'd still like to include an option to add prettier if possible seeing as we're not adding it to the eslint config itself.

As for the root: I've not usually used it, but it was in the example in our [docs](https://docs.expo.dev/guides/using-eslint/) so I added it. I don't have a strong opinion on it though.